src/components/NucleusCodeSnippet.js

```diff
@@ -41,12 +44,24 @@ export class NucleusCodeSnippet extends LitElement {
         htmlContent = `${htmlContentParts[0]}${prefixToLookup}
           ${(baseStyles || themedStyles) ? '<style>' : ''}
           ${baseStyles ?? ''}
+          ${`
+            .expressive-code .frame {
+              border-radius: ${borderRadiusStyles}
+            }
+            .expressive-code pre {
+              border-radius: ${borderRadiusStyles}
+            }
+            .expressive-code .copy button {
+              opacity: initial;
```

Comment:
Here is some CSS trickery to add the content from the `title` attribute to display 'Copy to clipboard'.

```suggestion
              opacity: initial;
              position: relative;
              border: 1px solid rgb(225 228 232 / .4);
              
              &::before {
                pointer-events: auto;
                border: none;
                content: attr(title);
                opacity: 1;
                color: rgb(225 228 232);
                width: max-content;
                top: 50%;
                transform: translate(min(calc(-100% - 0.5rem)), -50%);
              }
```

Reply:
I would like to keep the content short as 'Copy'.